web/cobrands/sass/_base.scss

```diff
@@ -1775,8 +1786,15 @@ label .muted {
   display: inline-block;
 }
 
-#problem_form > label:first-child {
-  margin-top: 0;
+#problem_form {
+  & > label:first-child {
+    margin-top: 0;
+  }
+
+  // Reduce space between labels and their form-hints
+  label + .form-hint {
```

Comment:
This is nice, should it be more general? e.g. for the one on `/alert`, or if you added similar to the update form, an admin form, or wherever.

Reply:
Good idea. Looks like we already have something similar (`margin-top: -0.25em`)on `#front-main #postcodeForm label + .form-hint` so I see whether I can get all these standardised.